src/collection-browser.ts

```diff
@@ -118,6 +120,8 @@ export class CollectionBrowser
 
   @property({ type: Boolean }) showHistogramDatePicker = false;
 
+  @property({ type: String }) collectionPagePath: string = '/details/';
```

Comment:
By default, links to other collections point at their /details/<id> page. This enables customizing that behavior, e.g. to make them point at /collection/<id> instead.